apps/recnet-api/src/modules/slack/transporters/slack.transporter.ts

```diff
@@ -26,11 +34,43 @@ export class SlackTransporter {
     @Inject(AppConfig.KEY)
     private readonly appConfig: ConfigType<typeof AppConfig>
   ) {
-    this.client = new WebClient(this.slackConfig.token);
+    this.client = new WebClient();
+  }
+
+  public async accessOauthInfo(
```

Comment:
Curious if it would be better to put this function at `slack.service.ts`. Any specific reason to put here?

Reply:
The original design philosophy is to make the transporter an internal service within the slack module that communicates with Slack external service and the SlackService is the service that serves the logic for RecNet requirements. I think in this case we can do either way, but to simplify the dependencies, I moved it to SlackService.

Reply:
Yeah I agree either works